packages/xml/src/decorators.ts

```diff
@@ -19,32 +19,46 @@ import type { XmlNamespace } from "./types.js";
 /** @internal */
 export const namespace = "TypeSpec.Xml";
 
+/**
+ * @public
```

Comment:
this rule force every exported item to have `@public`, `@beta`, `@alpha` or `@internal`

Reply:
but we could disable it if people feel like this is uncessary

Reply:
I only think it's useful until we have api view automated checks, since this will catch if you exported something unintentionally.